src/app/core/auth/state/auth.actions.ts

```diff
@@ -0,0 +1,27 @@
+import { Tokens } from 'src/app/core/auth/types/tokens'
+import { Credentials } from "../types/credentials";
+
+export class Login {
+  public static readonly type = '[Auth] Login'
+
+  constructor(public readonly credentials: Credentials, public readonly remember: boolean) {
+  }
```

Comment:
There's something wrong with your Prettier I think. This line is too long, and would be broke by formatter if it works.

Reply:
Configure it to automatically format on save,